docker/anet-mssql-linux/entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+if [ "$ACCEPT_EULA" -ne "Y" ]; then
+    echo "Need to accept MS SQL's EULA and set ACCEPT_EULA to Y"
+    exit 1
+fi  
+
+if [ -z "$SA_PASSWORD" ]; then
+    echo "Need to set SA_PASSWORD"
+    exit 1
+fi  
+
+mkdir -p /var/opt/mssql
+/opt/mssql/bin/sqlservr &
+
+#fixme to something better than a sleep
+sleep 10
+
+if [ -z "$DB_NAME" ]; then
+    echo "Need to set DB_NAME"
+    exit 1
+fi  
+
+if [ -z "$DB_USER" ]; then
+    echo "Need to set DB_USER"
+    exit 1
+fi  
+
+if [ -z "$DB_USER_PASSWORD" ]; then
+    echo "Need to set DB_USER_PASSWORD"
+    exit 1
+fi  
+
+cat <<-EOSQL > /var/opt/mssql/init.sql
+CREATE DATABASE $DB_NAME;
+GO
+USE $DB_NAME;
+GO
+CREATE LOGIN $DB_USER WITH PASSWORD = '$DB_USER_PASSWORD';
+GO
+CREATE USER $DB_USER FOR LOGIN $DB_USER;
+GO
+ALTER SERVER ROLE sysadmin ADD MEMBER [$DB_USER];
```

Comment:
Should be:
`ALTER SERVER ROLE sysadmin ADD MEMBER $DB_USER;`
without the square brackets I believe.